src/PlusOpenIGTLink/PlusIgtlClientInfo.h

```diff
@@ -63,21 +72,28 @@ struct vtkPlusOpenIGTLinkExport PlusIgtlClientInfo
   std::vector<PlusTransformName> TransformNames;
 
   /*! String field names to send with IGT STRING message */
-  std::vector< std::string > StringNames;
+  std::vector<std::string> StringNames;
 
   /*! Transform names to send with IGT image message */
   std::vector<ImageStream> ImageStreams;
 
-  /*! A new TDATA is only sent if the time elapsed is at least the resolution
-     value (otherwise we don't send this tracking data to the client) */
-  int Resolution;
+protected:
+  /*! IGTL header version supported by the client */
+  int ClientHeaderVersion;
+
+  /*! Flag for send blocking or not (default no) */

```

Comment:
This description is not clear. What does it do exactly?
Also, if there are public get/set functions then the value should be documented there.